puppet/zulip/manifests/common.pp

```diff
@@ -69,13 +69,14 @@
     # https://github.com/wal-g/wal-g/releases
     'wal-g' => {
       'version' => '1.1.3-rc-with-build',
-      'sha256' => {
+      'sha256'  => {
         'amd64' => '109a80f4c019e0f1d52602e90d2a181eb844494ece2d099a149cf9204b71113e',
-        # No aarch64 builds
+        # aarch64 builds from source, below
       },
+      # This is a Git commit hash, not a sha256sum, for when building from source.
+      'sha1'    => '52f990fe87679ee4651fe3fb7629a2ac799f50c6',
```

Comment:
Would it be better to name this `git_commit_for_build`?

Reply:
That seems a little wordy for me, and I don't want someone to put a tag name in there -- the goal is to validate the tag hasn't moved, not to be any ol' ref.

Reply:
`git_commit_id` would probably work.